listenbrainz_spark/hdfs/upload.py

```diff
@@ -36,7 +37,7 @@ def process_json_listens(self, filename, data_dir, tmp_hdfs_path, schema):
                 tmp_HDFS_path (str): HDFS path where listens JSON has been uploaded.
         """
         start_time = time.time()
-        df = utils.read_json(tmp_hdfs_path, schema=schema).cache()
+        df = utils.read_json(tmp_hdfs_path, schema=schema)
```

Comment:
why are we removing the cache function? I have some inkling that caching the file was needed for the import to work correctly because we delete the json later. Have you tested that the import works without this?

Reply:
>  Have you tested that the import works without this?

yes, for mappings (~ 6GB after decompression) , artist_relations (2GB after decompression) and listens (small set - incremental)